### PR TITLE
LPS-136149

### PR DESCRIPTION
**Pull request by mlan0**

### CODE REVIEW

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java [hunk 1]
```diff
@@ -204,6 +210,24 @@ private Object _getUserProfileImage(long userId) {
...
 	}
 
+	private Object _getValue(
+		ObjectField objectField, Map<String, Serializable> values) {
+
+		if (Objects.equals(
+				_getInfoFieldType(objectField), ImageInfoFieldType.INSTANCE)) {
+
+			try {
+				return JSONFactoryUtil.createJSONObject(
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java]: Hey Matheus, would you mind to use WebImage to follow the same pattern existing in other classes which implements InfoItemFieldValuesProvider interface?

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java (hunk 1 above)
Comment (mlan0) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java]: Done!

**Comment by mlan0** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

**Review by guilhermedcamacho — APPROVED** [review]

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

**Comment by guilhermedcamacho** [comment]
Sent directly to Brian in this [PR](https://github.com/brianchandotcom/liferay-portal/pull/106425)

**Pull request closed without merging**